src/pluggy/hooks.py

```diff
@@ -240,7 +240,13 @@ def _add_hookimpl(self, hookimpl):
             methods.insert(i + 1, hookimpl)
 
     def __repr__(self):
-        return "<_HookCaller %r>" % (self.name,)
+        return "<_HookCaller %r, _wrappers=%r, _nonwrappers=%r, _call_history=%r, spec=%r>" % (
```

Comment:
i would like to see a example on the effect this has when more than just one plugin is added to a setup

i believe its going to be a problem as proposed here

Reply:
See if the new version fits your liking